CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator.swift

```diff
@@ -233,6 +237,11 @@ extension GiniScreenAPICoordinator {
     }
 
     @objc func showHelpMenuScreen() {
+        //TODO: rethink this when you add Camera access screen events!
```

Comment:
[Clarifcation]: would you like to keep this inline comment?

Reply:
I would like to keep it until I add those events for the camera access screen. It will be much easier for me.